RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate SelectListView to Typescript

## Proposed changes

Migrate SelectListView to Typescript

## Issue(s)	


## How to test or reproduce

- Create a team and a channel
- Open the team room, go to `Channels View`, the `#`icon near the title, then click on `edit` icon, next to search icon
- The view `AddChannelToTeam` will show and click on `Add Existing`
- Here is the view migrated. So you must see the channel that you created before. Test the search too.

## Screenshots

## Types of changes




## Checklist



## Further comments
